doc/api/buffer.md

```diff
@@ -165,6 +165,38 @@ console.log(Buffer.from('fhqwhgads', 'utf16le'));
 // Prints: <Buffer 66 00 68 00 71 00 77 00 68 00 67 00 61 00 64 00 73 00>
 ```
 
+### Evaluating legal code points for '`utf-8'` encoding
```

Comment:
Should this subsection be placed at the end of the current section, before the `Buffers and TypedArray` section? Currently, this new subsection is continued with the "The character encodings currently supported by Node.js include..." part without any border or heading which can be confusing.